apps/rush-sdk/config/heft.json

```diff
@@ -0,0 +1,90 @@
+/**
+ * Defines configuration used by core Heft.
+ */
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/heft/heft.schema.json",
+
+  "extends": "@rushstack/heft-node-rig/profiles/default/config/heft.json",
+  "eventActions": [
+    {
+      /**
+       * (Required) The kind of built-in operation that should be performed.
+       * The "copyFiles" action copies files that match the specified patterns.
+       */
+      "actionKind": "copyFiles",
+
+      /**
+       * (Required) The Heft stage when this action should be performed.  Note that heft.json event actions
+       * are scheduled after any plugin tasks have processed the event.  For example, a "compile" event action
+       * will be performed after the TypeScript compiler has been invoked.
+       *
+       * Options: "pre-compile", "compile", "bundle", "post-build"
+       */
+      "heftEvent": "pre-compile",
+
+      /**
+       * (Required) A user-defined tag whose purpose is to allow configs to replace/delete handlers that
+       * were added by other configs.
+       */
+      "actionId": "copy-rush-lib-dts",
+
+      /**
+       * (Required) An array of copy operations to run perform during the specified Heft event.
+       */
+      "copyOperations": [
+        {
+          /**
+           * (Required) The base folder that files will be copied from, relative to the project root.
+           * Settings such as "includeGlobs" and "excludeGlobs" will be resolved relative
+           * to this folder.
+           * NOTE: Assigning "sourceFolder" does not by itself select any files to be copied.
+           */
+          "sourceFolder": "node_modules/@microsoft/rush-lib/dist/",
+
+          /**
+           * (Required) One or more folders that files will be copied into, relative to the project root.
+           * If you specify more than one destination folder, Heft will read the input files only once, using
+           * streams to efficiently write multiple outputs.
+           */
+          "destinationFolders": ["dist"],
+
+          /**
+           * If specified, this option recursively scans all folders under "sourceFolder" and includes any files
+           * that match the specified extensions.  (If "fileExtensions" and "includeGlobs" are both
+           * specified, their selections are added together.)
+           */
+          // "fileExtensions": [".jpg", ".png"],
+
+          /**
+           * A list of glob patterns that select files to be copied.  The paths are resolved relative
+           * to "sourceFolder".
+           * Documentation for supported glob syntaxes: https:www.npmjs.com/package/fast-glob
+           */
+          "includeGlobs": ["rush-lib.d.ts"]
```

Comment:
Currently `rush-sdk` depends on `rush-lib` because it needs to copy **rush-lib.d.ts**.  

We could eliminate this dependency, by adding a postbuild step for `rush-lib` that prints a STDERR warning if the files are not the same (similar to API Extractor's warning when .api.md is out of date).  Then the person is required to manually copy it over.  

Eliminating this dependency would enable `rush-lib` to depend on the built-in plugins, avoiding the need for a build step to fix up [publishOnlyDependencies](https://github.com/microsoft/rushstack/blob/7fc5f92a6a29afaf339fbc58c85d8cf0f4e924ae/apps/rush-lib/package.json#L83) during publishing.

Reply:
Done

Reply:
This idea did not work out, because the built-in plugins still need a `devDependency` on `rush-lib` for testing purposes.  Thus the `publishOnlyDependencies` seems unavoidable.

Reply:
However it does still seem useful to eliminate the `rush-sdk` dependency on `rush-lib`, since that way `rush-lib` can import an internal API from `rush-sdk`  for wiring up the proxy.

Reply:
...but then that requires tracking the entire `rush-lib.d.ts` file in Git.  I think I'm going to keep the original approach since it's overall simpler.